sec1_intro_and_setup/test_app/public/robots.txt

```diff
@@ -0,0 +1 @@
+# See https://www.robotstxt.org/robotstxt.html for documentation on how to use the robots.txt file
```

Comment:
Whats this file for?

Reply:
This file allows you to specify how search engines crawl your website.

Reply:
How does search engine use this file to crawl the site?

Reply:
Search engines use this file to categorize websites based on the robots exclusion protocols stated in this file. The protocols help the search engine robots to decide which areas of the website should not be processed or scanned to avoid overloading requests for the website.